### DISCUSSION TITLE
Creating a web page and a system for interacting with Clients.

### DISCUSSION THREAD

**Discussion by soustan** [Budibase/budibase#14313] [Help]
### Discussed in https://github.com/Budibase/budibase/discussions/14308

<div type='discussions-op-text'>

<sup>Originally posted by **soustan** August  3, 2024</sup>
Hello!
1. Do I understand correctly that with the help of budibase tools. I will be able to configure only the part of the development concerning the backend?
Or are there still tools that will allow you to configure both the backend and the frontend? However, I did not find such tools for the frontend part on this resource.
2. The fact is that I need to create, configure and subsequently administer an Internet resource. Which is a platform for interaction with Clients. Namely:
* Is a CRM system (exclusively based on open source code) for sales, maintenance, management of interaction with Clients, including the collection of information and data about Clients. Their needs for a particular product / client service, which is created by the developer of a particular web service;
* Issuing documents for payment, bills, invoices and other accounting documents.
* Uploading information about Clients from both the Client and open (official) sources into the database I am creating.
* Analysis of existing and new Clients for identity (affiliation) in other organizations. That is, the relationship between a particular Client.
I have several existing CRM systems in my work.
For example: https://salesap.ru, https://www.amocrm.ru or its analogue https://www.kommo.com . However, on the one hand, they are paid. And they require large and regular investments. However, in my opinion, the main thing is that all the data and all the invaluable and important information is stored somewhere there... And at any time, it may become unavailable to me.
Summarizing the above:
It is necessary to create an Internet resource, which is a page that collects and analyzes information about Clients. Creating LIDs, sending information to potential Clients, for subsequent sales, web services and related services to Clients. And also the creation of a system of interaction with third-party web services and other service providers. Communication with such providers is possible with the help of CRUD interactions (POSTMAN, TALENT and others).
Thus, I know what I ultimately need. However, I still do not understand in what way / methods I can achieve this?
Please understand my questions and the direction of my activities - right! I do not need advice on who to hire for this work. Or what I need to study, this and / or that ... Since I need to do everything described on my own. This is a fundamental question !!
And based on the information I read, the technology with a minimum of (LOW) code. Which budibase offers, allows me to achieve my ultimate interests described above.
I ask for help and recommendations from competent specialists and colleagues.</div>

**Comment by linear** [comment]
<p><a href="https://linear.app/budibase/issue/BUDI-8537/creating-a-web-page-and-a-system-for-interacting-with-clients">BUDI-8537 Creating a web page and a system for interacting with Clients.</a></p>
